forward_engineering/helpers/general.js

```diff
@@ -86,6 +86,31 @@ module.exports = app => {
 		return options.length ? `\n OPTIONS(\n${tab(options.join(',\n'))}\n)` : '';
 	};
 
+	const foreignKeysToString = keys => {
```

Comment:
Please use `foreignKeysToString` and `foreignActiveKeysToString` from `app.require('@hackolade/ddl-fe-utils').general`